### PR TITLE
Release notes for 4.13.2S and 4.13.3S added

### PR DESCRIPTION
**Pull request by Hikkigaya**
### Summary

Release notes for 4.13.2S and 4.13.3S added

Preview links
- http://10.2.7.208/release-notes/standalone-kobiton-four-latest
- http://10.2.7.208/release-notes/all-releases/4_13_2S

### Related PRs, issues, or features (optional)

- 247

### Metadata

- [x] Adds new file(s)
- [x] Edits existing file(s)
- [ ] Removes file(s)

### PR contributor checklist

- [x] My PR follows the [Kobiton Docs contributor guidelines](https://github.com/kobiton/docs/blob/main/CONTRIBUTE.adoc), meaning:

    - My content contains correct spelling and grammar.
    - My directories and files are [named](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-and-file-names) and [structured](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-structure) correctly.
    - My style and voice follows the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human).
    - I added all new pages to their section's [`nav.adoc` file](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#configure-navigation-in-navadoc).
    - I removed all localizations (like `en-us`) from my URLs.

### CODE REVIEW

**Review by tungmhoang — CHANGES_REQUESTED** [review]
@Hikkigaya when we add a new release notes page, we need to also update the reference in the `latest` pages. In this case, it it the `docs/modules/release-notes/pages/standalone-kobiton-four-latest.adoc` page.
Please help to update the reference link to the 4.13.2S release notes.

File: docs/modules/release-notes/pages/all-releases/4_13_2S.adoc [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+* iOS/iPadOS 17.4.1, 17.5 Beta, 17.5.1
+* Mac OS Sonoma 14.4.1, 14.5
+* Android 15 Beta 3
+
+[NOTE]
+====
+* At this time, we recommend hosting iOS 17+ devices via Standard mode deployments for improved performance.
+* For Android 15 Beta 3, the cleanup policy cannot remove a signed-in account.
+* For iOS/iPadOS 18 Beta, network payload capture is not supported.
```
Comment (tungmhoang) [inline docs/modules/release-notes/pages/all-releases/4_13_2S.adoc:16]: @Hikkigaya Remove this line as iOS 18 Beta is not yet supported.

File: docs/modules/release-notes/pages/all-releases/4_13_2S.adoc [hunk 2]
```diff
@@ -0,0 +1,115 @@
+= Kobiton 4.13.2S release notes
+:navtitle: Kobiton 4.13.2S release notes
+
```
Comment (tungmhoang) [inline docs/modules/release-notes/pages/all-releases/4_13_2S.adoc:3]: Please add the release date like other release notes. For this release, the date is June 20, 2024

**Review by tungmhoang — CHANGES_REQUESTED** [review]

File: docs/modules/release-notes/pages/all-releases/4_13_2S.adoc (hunk 1 above)
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_2S.adoc:16]: Changes made

File: docs/modules/release-notes/pages/all-releases/4_13_2S.adoc (hunk 2 above)
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_2S.adoc:3]: Changes made.

**Review by tungmhoang — CHANGES_REQUESTED** [review]
@Hikkigaya could you add another release notes (attached) into this PR as well. Please also replace the xref in the latest standalone release notes to this one.
[KOBITON-v4.13.3S July 19, 2024 [Review]-260724-135426.pdf](https://github.com/user-attachments/files/16392706/KOBITON-v4.13.3S.July.19.2024.Review.-260724-135426.pdf)

**Review by tungmhoang — APPROVED** [review]

**Comment by tungmhoang** [comment]
@erincbailey you can review

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+== New OS support
+
+kobiton 4.13.3S add support for the following:
+
+* iOS/iPadOS 18 Beta
+* Xcode 16 Beta
+
+== CoreDevice: Lightning mode support for iOS 17.4 devices and later without networking changes
+
+When Apple introduced some major changes with the rollout of iOS 17, it requires significant network configuration to support Lightning mode with Kobiton. The additional network requirements are hard to implement and sometimes pose a security risks for some organizations.
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:15]: Please replace this paragraph with the following:
When Apple introduced iOS 17, they rolled out new connectivity protocols that impacted how iOS 17 devices communicated with solutions like Kobiton. These protocols required special network configurations for Kobiton Lightning Mode deployments that were not always possible for customers to implement.

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+With this release, we are happy to introduce CoreDevice, which is a solution to the above problem. With CoreDevice, iOS 17.4 and later devices can have Lightning mode without the additional network requirements.
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:17]: Replace this sentence with: 
With iOS 17.4, Apple introduced the CoreDevice communication protocol that significantly enhances connections between iOS devices and the Kobiton platform. We implemented this innovative protocol with this release. CoreDevice eliminates the need for the special Lightning Mode networking changes introduced with iOS 17 for iOS 17.4+ devices, simplifying your networking.

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+With iOS 17.0 to 17.3 devices, the necessary components to support CoreDevice is not available. We recommend hosting these devices in Standard mode with the additional Mac mini instead.
+
+[NOTE]
+====
+
+* iOS 17.4 and later devices still need to be plugged into the Mac  mini to establish trust before connecting to the GEM.
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:24]: There looks like an extra space before "mini"

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+* iOS/iPadOS 18 Beta
+* Xcode 16 Beta
+
+== CoreDevice: Lightning mode support for iOS 17.4 devices and later without networking changes
+
+When Apple introduced some major changes with the rollout of iOS 17, it requires significant network configuration to support Lightning mode with Kobiton. The additional network requirements are hard to implement and sometimes pose a security risks for some organizations.
+
+With this release, we are happy to introduce CoreDevice, which is a solution to the above problem. With CoreDevice, iOS 17.4 and later devices can have Lightning mode without the additional network requirements.
+
+With iOS 17.0 to 17.3 devices, the necessary components to support CoreDevice is not available. We recommend hosting these devices in Standard mode with the additional Mac mini instead.
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:19]: Replace above with: 
Apple's CoreDevice cannot be used with iOS 17.0-17.3 devices. These devices must be hosted on a separate Mac mini in Standard mode.

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+_July 19, 2024_
+
+== New OS support
+
+kobiton 4.13.3S add support for the following:
+
+* iOS/iPadOS 18 Beta
+* Xcode 16 Beta
+
+== CoreDevice: Lightning mode support for iOS 17.4 devices and later without networking changes
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:13]: Please take out CoreDevice: from the title of this section. Start with "Lightning mode...."

**Review by erincbailey — CHANGES_REQUESTED** [review]
A few comments. Apologies I hadn't reviewed the CoreDevice section before it was added here.

File: docs/modules/release-notes/pages/all-releases/4_13_2S.adoc [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+_Prerequisites/Limitations_:
+
+* iOS/iPadOS 14 or later, Xcode 14 or later.
+* Download the Kobiton custom camera framework.
+* Access to the application’s source code (to implement custom library).
+* There is no indicator that an app has the Kobiton camera framework implemented in the app repo.
+* Inject the image after launching the camera. Doing this before launching the camera might crash the app.
+* It may take up to 5 seconds for the injected image to appear in the camera view. Be patient!
+
+A detailed user guide for the image injection SDK for iOS is xref:apps:image-injection-sdk/add-the-sdk-to-your-ios-app.adoc[now available].
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_2S.adoc]: Changed

File: docs/modules/release-notes/pages/all-releases/4_13_2S.adoc [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+_Prerequisites/Limitations_:
+
+* Image injection requirements for iOS and Android apply.
+* Using the `'kobiton:instrumentenabled'` capability to instrument the app for camera injection does not work if in the `dc.ini` file of the Mac mini host, the value of `'Kobiton.UseInternalLegacyAppResigning'` is `false`. Change this value to `true` to fix the issue.
+* Only supported in Xium automation sessions.
+* The image must be located on the runner machine and be passed with `base64` encoding. Passing the image URL is not yet supported. Inject the image after launching the camera.
+* Injecting the image before launching the camera might crash the app.
+* Add a wait time for the image injection process to complete (up to 5 seconds) before performing the next command.
+
+A user guide is xref:automation-testing:scripting/add-image-injection-to-appium-script.adoc[now available].
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_2S.adoc]: Removed

File: docs/modules/release-notes/pages/all-releases/4_13_2S.adoc [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+* Addressed the following Script-based test automation and Device Inspector items:
+** Add actions, screenshots, and HTTP headers in Session Explorer for Basic Appium 2 sessions.
+** Fix an issue where `sendKeys` method cannot send text to password fields in iOS.
+** Fix an issue with Chrome WebView being returned instead of the desired application view in sessions with Android hybrid apps.
+** Fix an issue with missing inspection data for a 4-5MB XML tree.
+** Exclude some unused attributes in the XML of inspector data.
+** Make the `deviceOrientation` desired capability work as intended in the automation session.
+** Return `resourceId` in Device Inspector.
+** New capability `'kobiton:xpath2'` to control the Xpath version used. (Use Xpath 2.0 when `true` ).
+** Fixed the error `kAXErrorServerNotFound` that prevents the Device Inspector from finding elements. This fixes issues for both script based automation and Passcode devices.
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_2S.adoc]: Added

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 11]
```diff
@@ -0,0 +1,107 @@
+= Kobiton 4.13.3S release notes
+:navtitle: Kobiton 4.13.3S release notes
+
+_July 19, 2024_
+
+== New OS support
+
+kobiton 4.13.3S add support for the following:
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc]: Done

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+* Clear indications of the unsupported actions are displayed in test steps.
+* All steps containing unsupported actions must be removed from a test case before test runs can be triggered.
+
+=== Things to note:
+As this feature is in Beta, we welcome your feedback. Stay tuned for additional UI improvements and extended functionality in the coming months. We will publish a comprehensive User Guide on our documentation site soon. In the meantime, please note the following:
+
+* Launching Test Runs: test runs can be launched via:
+** The New Test Run button in the test run list.
+** The Run button in test case and test suite pages to make it easier to initiate a test run.
+** The Rerun button to the test run page. When selected, the configuration of the new test run will be pre-filled, including the specific device/device bundle previously used.
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:69]: Changed

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+
+* Testing Capabilities: With the Beta release, only crash and accessibility validations are available. We are actively developing performance, text/color, and visual validations. Data-driven testing is not supported in the Beta, but will be added in a future release.
+
+* Test Run Statuses:
+** Running: The test run session is either pending execution or in progress.
+** Needs Action: A blocker has been encountered, and remediation is required.
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc]: Removed

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc (hunk 3 above)
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc]: Replaced

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc (hunk 5 above)
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc]: Removed

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc (hunk 6 above)
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc]: Replaced

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc (hunk 7 above)
Comment (Hikkigaya) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc]: Removed

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+Kobiton 4.13.3S add support for the following:
+
+* iOS/iPadOS 18 Beta
+* Xcode 16 Beta
+
+== Lightning mode support for iOS 17.4 devices and later without networking changes
+
+When Apple introduced iOS 17, they rolled out new connectivity protocols that impacted how iOS 17 devices communicated with solutions like Kobiton. These protocols required special network configurations for Kobiton Lightning Mode deployments that were not always possible for customers to implement.
+
+With iOS 17.4, Apple introduced the CoreDevice communication protocol that significantly enhances connections between iOS devices and the Kobiton platform.We implemented this innovative protocol with this release. CoreDevice eliminates the need for the special Lightning Mode networking changes introducedwith iOS 17 for iOS 17.4+ devices, simplifying your networking.
```
[the remark this answers is not in the document]
Comment (erincbailey) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:17]: Also need a space after platform.

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+
+Apple's CoreDevice cannot be used with iOS 17.0-17.3 devices. These devices must be hosted on a separate Mac mini in Standard mode.
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:19]: Change to: 
NOTE: iOS 17.0-17.3 devices continue to require special networking to support Lightning mode, so we recommend those devices are hosted on a separate Mac mini in Standard mode.

Comment (tungmhoang) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc]: updated

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc (hunk 14 above)
[the remark this answers is not in the document]
Comment (tungmhoang) [reply docs/modules/release-notes/pages/all-releases/4_13_3S.adoc]: updated

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 16]
```diff
@@ -0,0 +1,112 @@
...
+* Test Run Statuses:
+** Running: The test run session is either pending execution or in progress.
+** Needs Action: A blocker has been encountered and remediation is required.
+** Error: A system error has been encountered.
+** Success: The test run completed.
+** Failed: The application crashed during the test run.
+
+* Search and filters:
+** Filter test runs, test suites, and test cases by name or app bundle name. Note: Hit Enter after inputting the search query to start filtering.
+** Filter test suites and test runs by a single test case by selecting the number of test runs/test suites in the test case list
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:88]: Need a period at end to be consistent with first bullet. Or need to remove period from first bullet

File: docs/modules/release-notes/pages/all-releases/4_13_3S.adoc [hunk 17]
```diff
@@ -0,0 +1,112 @@
...
+
+* Blocker Remediations: When a Blocker is encountered for a Touch action during a test run, you can now remediate the step where the Blocker happened or review and remediate in a previous step. The submitted remediation will apply to subsequent reruns from the same test case and on the same device model.
+
+* Create Test Case from Appium script: We do not currently support creating a Test Case from an Appium script using `scriptless:enable`  with this Beta version. We will reintroduce that feature in an upcoming release.
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_13_3S.adoc:92]: Is there an extra space after scriptless:enable?

File: docs/modules/release-notes/pages/all-releases/4_13_2S.adoc [hunk 18]
```diff
@@ -0,0 +1,118 @@
...
+====
+
+== Combined HAR files for network payload data
+
+Previously, the network payload capture log was split into multiple .payload files. We can now consolidate multiple network log files into a single payload file, making it easier to convert to load test scripts on products like Grafana K6, JMeter, and Neoload for performance testing.
+
+We also introduced two options for downloading the payload data:
+
+* Include both request and response payloads.
+* Include request payloads only (reduced size and optimized for load testing)
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_13_2S.adoc:27]: Need to remove period in first bullet or add period in second bullet to be consistent

**Review by erincbailey — CHANGES_REQUESTED** [review]
Just a few minor edits. You can publish once done. I don't need to review again.

**Review by tungmhoang — APPROVED** [review]

**Comment by tungmhoang** [comment]
All feedbacks addressed. Will merge.

**Pull request merged**